AUTHORS

```diff
@@ -123,15 +135,18 @@ Koichiro Den                            den@klaipeden.com
 Kornilios Kourtis                       kornilios@isovalent.com
 Laurent Bernaille                       laurent.bernaille@datadoghq.com
 Lehner Florian                          dev@der-flo.net
+Lior Rozen                              liorrozen@users.noreply.github.com
 Liu Qun                                 qunliu@zyhx-group.com
 Li Yi                                   denverdino@gmail.com
 m4rx0                                   m@footek.ch
+Maciej Fijalkowski                      maciej.fijalkowski@intel.com
 Maciej Kwiek                            maciej@isovalent.com
 Maciej Skrocki                          maciejskrocki@google.com
 Madhu Challa                            madhu@cilium.io
 MaiReo                                  sawako.saki@gmail.com
 Maksym Lushpenko                        iviakciivi@gmail.com
 Manali Bhutiyani                        manali@covalent.io
+mandarjog                               mandarjog@gmail.com
```

Comment:
Might want to add a `.mailmap` entry with the name as well, see 53f35fbdd7dd730a496796665fd9c8730678efe2:

```
Mandar U Jog <mjog@google.com> <mandarjog@gmail.com>
```